data/src/main/java/com/teamagam/gimelgimel/data/common/OkHttpClientFactory.java

```diff
@@ -50,43 +49,40 @@ private static Proxy createProxy() {
   }
 
   private static HostnameVerifier createHostnameVerifier() {
-    return new HostnameVerifier() {
-      @Override
-      public boolean verify(String hostname, SSLSession session) {
-        try {
-          if (session.getPeerCertificates()[0] instanceof X509Certificate) {
-            X509Certificate x509Cer = (X509Certificate) session.getPeerCertificates()[0];
-            String compare = Constants.PROXY_HOST;
-            String subject = x509Cer.getSubjectDN().getName();
-            String[] splitParts = subject.trim().split(",");
-            String[] subPart;
-            for (int i = 0; i < splitParts.length; i++) {
-              subPart = splitParts[i].split("=");
-              if (subPart[0].equals("CN") && subPart[1].compareToIgnoreCase(compare) == 0) {
-                return true;
-              }
+    return (hostname, session) -> {
```

Comment:
Sorry. I like the new one. In addition, the IDE is to my side. :stuck_out_tongue: